content/en/docs/Performance/_index.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "Performance"
+weight: 150
+date: 2020-11-20
+description: >
+  Baker performance test cases
+---
+
+We report in this page some practical examples of Baker performances we could measure in the
+NextRoll production environment.
+
+Within NextRoll, Baker is often executed on AWS EC2 instances, and thus you find in this page
+many references to
+[EC2 instance types](https://aws.amazon.com/ec2/instance-types/) (`c5.2xlarge`, `c5.2xlarge`, etc.).
+
+### Read from S3 and write to local disk
+
+On an AWS EC2 instance of size `c5.2xlarge`, Baker can read zstandard records from S3, uncompress
+them and apply a basic filtering logic, compressing them back on local files with zstandard at
+compression level 3 and long range mode at 27 using ~90% of capacity of each vCPU (8 in total) and
```

Comment:
I wouldn't write long range mode without explaining what that means here since it's very technical and very specific.
But I'm not sure we should write that, nor compression level 3 for that matter.

Reply:
removed